vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -18,8 +18,10 @@
           "1.22.6-hotfix.20220130",
           "1.22.6-hotfix.20220310.1",
           "1.23.3-hotfix.20220130.2",
-          "1.23.3-hotfix.20220310.2",
-          "1.23.4.1"
+          "1.23.3-hotfix.20220401.1",
+          "1.23.4.1",
```

Comment:
Did not remove older version, in case the hotfix version has an issue detected in RP and we need to use the previous version (following the VHD release for this week).